jenkins-master/jobs/mozilla-aurora_functional/config.xml

```diff
@@ -89,24 +89,33 @@
       <selector class="hudson.plugins.copyartifact.StatusBuildSelector"/>
     </hudson.plugins.copyartifact.CopyArtifact>
     <hudson.plugins.xshell.XShellBuilder plugin="xshell@0.9-SNAPSHOT">
-      <commandLine>python runtests.py --type=functional --branch=mozilla-aurora --build-revision=$REVISION --build-locale=$LOCALE --installer-url=$INSTALLER_URL</commandLine>
+      <commandLine>python -u submission.py --test-type=functional --state=running --repository=mozilla-aurora --revision=$REVISION --locale=$LOCALE treeherder_venv</commandLine>
+      <executeFromWorkingDir>false</executeFromWorkingDir>
+    </hudson.plugins.xshell.XShellBuilder>
+    <hudson.plugins.xshell.XShellBuilder plugin="xshell@0.9-SNAPSHOT">
+      <commandLine>python runtests.py --type=functional --installer-url=$INSTALLER_URL</commandLine>
```

Comment:
For this PR I cannot test this because its not mozharness yet. But the mozharness script should not exit with 0 if marionette fails but should reflect its exit value. I will check this with the next big PR which will include the mozharness inclusion.